controllers/designer/kaoto_controller_action_deploy.go

```diff
@@ -120,6 +131,42 @@ func (a *deployAction) deploy(ctx context.Context, rr *ReconciliationRequest) er
 				},
 			}
 
+			// it appears that even with the standalone mode, there are some CORS issues that prevent some endpoint
+			// to be invoked, the QUARKUS_HTTP_CORS_ORIGINS env must be set.
+			//
+			// TODO: investigate why this is needed
```

Comment:
This is needed since the port and protocol exposed by the container differ from the exposed from the Openshift Route. More information [in this zulip thread](https://quarkusio.zulipchat.com/#narrow/stream/187030-users/topic/.E2.9C.94.20CORS.20.2F.20Some.20endpoints.20failing.20while.20others.20not/near/378094268).